CHANGELOG.md

```diff
@@ -7,7 +7,7 @@
 
 - Added support for `delimiters` parameter in `functions.initcap()`.
 - Added support for `functions.hash()` to accept a variable number of input expressions.
-- Flattened generated SQL when `DataFrame.filter()` is followed by a projection statement (e.g. `DataFrame.select()`, `DataFrame.with_column()`).
+- Flattened generated SQL when `DataFrame.filter()` of `DataFrame.order_by()` is followed by a projection statement (e.g. `DataFrame.select()`, `DataFrame.with_column()`).
```

Comment:
nit: `of` -> `or`?

Reply:
Oh this is a typo, thank you for catching!